docs/devel/hello-world-gadget.md

```diff
@@ -75,12 +76,21 @@ GADGET_TRACER_MAP(events, 1024 * 256);
 This macro will automatically create a ring buffer if the kernel supports it.
 Otherwise, a perf array will be created.
 
-And mark this map as a tracer map, i.e. a map that is used to push events to user space:
+Optionally, you can employ the `GADGET_TRACER` macro to define the gadget as a tracer,
+specifying its behaviour with the following parameters:
```

Comment:
I don't think we should say "define the gadget as a tracer" as very soon we'll support multiple tracers and even multiple kind of output in the same gadget.

```suggestion
Optionally, you can employ the `GADGET_TRACER` macro to define a tracer with the following parameters:
```